frontend/uddhar-frontend/src/coordinator/components/UserTable.jsx

```
@@ -53,19 +53,23 @@ const UserTable = () => {
                     </p>
                   </td>
                   <td>
-                    <button className="border-2 rounded-[5px] bg-green-200 p-1 cursor-pointer text-green-500 mr-1">
-                      ✔️
+                    <button className="border-2 rounded-[5px] bg-green-400 px-2 py-1 cursor-pointer text-black mr-1">
+                      ✓
                     </button>
-                    <button className="border-2 rounded-[5px] bg-red-200 p-1 cursor-pointer text-red-500">
-                      ❌
+                    <button className="border-2 rounded-[5px] bg-red-400 p-1 cursor-pointer text-black">
+                      ✖
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)